src/xrpld/app/tx/detail/LoanManage.cpp

```diff
@@ -223,11 +223,13 @@ LoanManage::defaultLoan(
         }
         if (*vaultAvailableProxy > *vaultTotalProxy)
         {
-            JLOG(j.warn()) << "Vault assets available must not be greater "
-                              "than assets outstanding. Available: "
-                           << *vaultAvailableProxy
-                           << ", Total: " << *vaultTotalProxy;
-            return tecLIMIT_EXCEEDED;
+            // LCOV_EXCL_START
+            JLOG(j.fatal())
+                << "Vault assets available must not be greater "
+                   "than assets outstanding. Available: "
+                << *vaultAvailableProxy << ", Total: " << *vaultTotalProxy;
+            return tecINTERNAL;
+            // LCOV_EXCL_STOP
```

Comment:
I wasn't sure this was a valid change, so I dug into it, and updated some tests. To my surprise, I discovered that fixing all the other rounding issues (so far) has made this condition essentially impossible, so `tecINTERNAL` is the appropriate result code.